frontend/model/state.js

```diff
@@ -361,8 +362,10 @@ const actions = {
         // this must be called directly, instead of via enqueueHandleEvent
         await dispatch('handleEvent', GIMessage.deserialize(events[i]))
       }
-      sbp('okTurtles.events/emit', EVENTS.CONTRACT_IS_SYNCING, contractID, false)
+    } else {
+      console.log(`Contract ${contractID} was already synchronized`)
```

Comment:
`console.debug`

Reply:
If this is `debug`, should the other logs on this fn also be renamed to `debug`?